research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-menu: Components
+menu: Proposals
```

Comment:
This was an error currently on the live site. I updated here it while I was in the code

<img width="269" alt="image" src="https://user-images.githubusercontent.com/17087167/158899549-50be392e-b8b4-4716-b439-d733f593842f.png">